tests/functional/repl.sh

```diff
@@ -25,6 +25,13 @@ import $testDir/undefined-variable.nix
 
 TODO_NixOS
 
+# FIXME: repl tests fail on systems with stack limits
+stack_ulimit="$(ulimit -Hs)"
+stack_required="67108864" # 64 * 1024 * 1024
```

Comment:
```suggestion
stack_required="$((64 * 1024 * 1024))"
```

I think something like that would work?